Test suite failures: Cannot find module slow, jshashes, spacetime, compromise

[Edit: I sent a fix in #369.]

Hi there,

Thanks for this fantastic library! I'm trying to run the test suite in preparation for sending a PR, but it's failing for me on master. I'm running Node 14.3.0 with npm 6.14.5.

Am I doing something wrong?

```
$ npm run test

> wtf_wikipedia@8.2.1 test /home/ubuntu/src/wtf_wikipedia
> TESTENV=dev node ./scripts/test.js



•••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••
   5.54s
   2,753  ✔️
category


internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module 'slow'
Require stack:
- /home/ubuntu/src/wtf_wikipedia/plugins/category/src/index.js
- /home/ubuntu/src/wtf_wikipedia/plugins/category/tests/_lib.js
- /home/ubuntu/src/wtf_wikipedia/plugins/category/tests/misc.test.js
- /home/ubuntu/src/wtf_wikipedia/node_modules/tape/bin/tape
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (/home/ubuntu/src/wtf_wikipedia/plugins/category/src/index.js:1:14)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/ubuntu/src/wtf_wikipedia/plugins/category/src/index.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/category/tests/_lib.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/category/tests/misc.test.js',
    '/home/ubuntu/src/wtf_wikipedia/node_modules/tape/bin/tape'
  ]
}

   0.13s
   0  ✔️
classify


•••••••••••••••••••••••••
   0.65s
   25  ✔️
html


••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••
   1.74s
   78  ✔️
i18n


•••••••••••
   0.20s
   11  ✔️
image


internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module 'jshashes'
Require stack:
- /home/ubuntu/src/wtf_wikipedia/plugins/image/src/url-hash.js
- /home/ubuntu/src/wtf_wikipedia/plugins/image/src/index.js
- /home/ubuntu/src/wtf_wikipedia/plugins/image/tests/_lib.js
- /home/ubuntu/src/wtf_wikipedia/plugins/image/tests/image.test.js
- /home/ubuntu/src/wtf_wikipedia/node_modules/tape/bin/tape
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (/home/ubuntu/src/wtf_wikipedia/plugins/image/src/url-hash.js:1:16)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/ubuntu/src/wtf_wikipedia/plugins/image/src/url-hash.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/image/src/index.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/image/tests/_lib.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/image/tests/image.test.js',
    '/home/ubuntu/src/wtf_wikipedia/node_modules/tape/bin/tape'
  ]
}

   0.13s
   0  ✔️
latex


•••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••
   1.58s
   77  ✔️
markdown


••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••
   1.61s
   80  ✔️
person


internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module 'spacetime'
Require stack:
- /home/ubuntu/src/wtf_wikipedia/plugins/person/src/getSentence.js
- /home/ubuntu/src/wtf_wikipedia/plugins/person/src/birthDate/index.js
- /home/ubuntu/src/wtf_wikipedia/plugins/person/src/index.js
- /home/ubuntu/src/wtf_wikipedia/plugins/person/tests/_lib.js
- /home/ubuntu/src/wtf_wikipedia/plugins/person/tests/birthDate.test.js
- /home/ubuntu/src/wtf_wikipedia/node_modules/tape/bin/tape
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (/home/ubuntu/src/wtf_wikipedia/plugins/person/src/getSentence.js:1:19)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/ubuntu/src/wtf_wikipedia/plugins/person/src/getSentence.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/person/src/birthDate/index.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/person/src/index.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/person/tests/_lib.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/person/tests/birthDate.test.js',
    '/home/ubuntu/src/wtf_wikipedia/node_modules/tape/bin/tape'
  ]
}

   0.14s
   0  ✔️
summary


internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module 'compromise'
Require stack:
- /home/ubuntu/src/wtf_wikipedia/plugins/summary/src/index.js
- /home/ubuntu/src/wtf_wikipedia/plugins/summary/tests/_lib.js
- /home/ubuntu/src/wtf_wikipedia/plugins/summary/tests/stress.test.js
- /home/ubuntu/src/wtf_wikipedia/node_modules/tape/bin/tape
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (/home/ubuntu/src/wtf_wikipedia/plugins/summary/src/index.js:1:13)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/ubuntu/src/wtf_wikipedia/plugins/summary/src/index.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/summary/tests/_lib.js',
    '/home/ubuntu/src/wtf_wikipedia/plugins/summary/tests/stress.test.js',
    '/home/ubuntu/src/wtf_wikipedia/node_modules/tape/bin/tape'
  ]
}

   0.12s
   0  ✔️
wikitext


•••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••
   1.61s
   77  ✔️
```

--- a/package.json
+++ b/package.json
@@ -19,6 +19,7 @@
     "build:all": "node ./scripts/buildAll.js",
     "start": "node ./scripts/demo.js",
     "coverage": "node scripts/coverage.js",
+    "prepare": "for i in plugins/*/; do (cd \"$i\" && npm install) || exit 1; done",
     "test": "TESTENV=dev node ./scripts/test.js",
     "testb": "TESTENV=prod node ./scripts/test.js",
     "test:fetch": "TESTENV=dev tape ./tests/fetch/*.test.js | tap-dancer",

